common/tools/dev-tool/src/commands/samples/publish.ts

```diff
@@ -173,7 +173,7 @@ async function processSources(
             if (tag.tagName.text === "summary") {
               log.debug("Found summary tag on node:", node.getText(sourceFile));
               // Replace is required due to multi-line splitting messing with table formatting
-              summary = tag.comment?.replace(/\s*\n\s*/, " ");
+              summary = tag.comment?.replace(/\s*\r?\n\s*/g, " ");
```

Comment:
Suggested fix by @witemple-msft. 
This was to fix the incorrect line endings that have overflown the text in the readme.

Reply:
I'm just cursed to get this wrong with JS regexp on the first try every time.